web/api/v1/api.go

```diff
@@ -933,10 +937,12 @@ func (api *API) rules(r *http.Request) apiFuncResult {
 
 	for i, grp := range ruleGroups {
 		apiRuleGroup := &RuleGroup{
-			Name:     grp.Name(),
-			File:     grp.File(),
-			Interval: grp.Interval().Seconds(),
-			Rules:    []rule{},
+			Name:           grp.Name(),
+			File:           grp.File(),
+			Interval:       grp.Interval().Seconds(),
+			Rules:          []rule{},
+			EvaluationTime: grp.GetEvaluationDuration().Microseconds(),
+			LastEvaluation: time.Since(grp.GetEvaluationTimestamp()).String(),
```

Comment:
We'd want to avoid exporting relative durations since the last evaluation here, but just the absolute last evaluation timestamp (like we do for scrapes of targets):

```suggestion
			LastEvaluation: grp.GetEvaluationTimestamp(),
```

...and then make the `LastEvaluation` field a `time.Time` type instead of `string`. Take a look at `LastScrape` and `LastScrapeDuration` for targets, we'll want to use the same existing pattern for rules.

Reply:
should I do the same for the `alertingRules` and `recordingRules` types?